src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -82,7 +81,6 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         model.setPerson(personToEdit, editedPerson);
-        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
```

Comment:
Why delete this line? We want the middle panel to be updated when user edits a contact.

Reply:
The middle panel will be updated without this line. But this line causes the contacts shown to jump back to the full contact list after an edit, while in (ab)find command, rather than the filtered contact list.